mappings/net/minecraft/client/gui/widget/ScrollableWidget.mapping

```diff
@@ -0,0 +1,46 @@
+CLASS net/minecraft/class_7528 net/minecraft/client/gui/widget/ScrollableWidget
+	COMMENT A widget that can be focused and vertically scrolled.
+	FIELD field_39493 FOCUSED_BORDER_COLOR I
+	FIELD field_39494 UNFOCUSED_BORDER_COLOR I
+	FIELD field_39495 BOX_COLOR I
+	FIELD field_39496 PADDING I
+	FIELD field_39497 scrollY D
+	FIELD field_39498 scrollbarDragged Z
+	METHOD method_44381 getPadding ()I
+	METHOD method_44382 setScrollY (D)V
+		ARG 1 scrollY
+	METHOD method_44383 isVisible (II)Z
+		ARG 1 top
+		ARG 2 bottom
+	METHOD method_44384 renderMiscellaneous (Lnet/minecraft/class_4587;)V
+		COMMENT Renders miscellaneous contents that are not scrolled but part of the widget.
```

Comment:
Maybe renderOverlay? I assume it's always done after the scrollable contents

Reply:
That sounds better.